source/js/main.js

```diff
@@ -7,16 +7,16 @@ import {
   bindCommonEventHandlers,
   GoogleAnalytics,
   initializePrimaryNav,
-  injectCommonReactComponents,
-  ReactGA
+  injectCommonReactComponents
 } from "./common";
 
-import { bindEventHandlers, injectReactComponents } from "./foundation";
+import {
+  bindWindowEventHandlers,
+  bindEventHandlers,
+  injectReactComponents
+} from "./foundation";
 
 import primaryNav from "./primary-nav.js";
-import bindMozFestGA from "./mozfest-ga.js";
-import bindMozFestEventHandlers from "./mozfest-event-handlers";
```

Comment:
MozFest scripts have been moved to `source/js/foundation/pages/mozfest` & the script is included in `mozfest-base.html` as additional script

https://github.com/mozilla/foundation.mozilla.org/pull/4443/files#diff-18ce4f535a3e43aee45bd880b861bbed